src/components/models/MiddleBlock.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
Is this block going to be used unchanged on multiple different pages? Just wondering why it's in `/models`

Reply:
Oh you right, I will create a new folder for `companiesPage`.
I put it here because Model include Companies